src/components/Layout/LinesReturn.tsx

```diff
@@ -103,6 +103,8 @@ export const LinesReturn = ({
     <DataTypeLines label='Return' dataType={describedReturnVal} />
   ) : isDelegated && !abi ? (
     <Line label='Return (upload abi to decode)'>{returnVal}</Line>
+  ) : isDelegated ? (
+    <NullishLine label='Return' />
```

Comment:
@Schwartz10 I'm not sure why we should add the `NullishLine` here? If the actor is delegated and we do have the ABI, then it should either parse to a `describedReturnVal` or we should show that it failed to decode with the raw `returnVal`. Showing a `NullishLine` just makes it seem like there is not return value altogether.